src/server.js

```diff
@@ -1,20 +1,16 @@
-require("dotenv").config();
-const logger = require("./config/logger");
+const bots = require("./config/robots.json").bots;
 
-const app = require("./app");
-const http = require("http");
+const GenericBot = require("./bot/genericBot")
 
-const roboter = require("./roboter/genericBoter");
+for (let i = 0; i < bots.length; i++) {
+  startBot(bots[i]);
+}
 
-var port = process.argv[2] === "DEBUG" ? 81 : process.env.PORT;
-
-http.createServer(app).listen(port, function () {
-  console.log("App listens on port: " + this.address().port);
-
-  try {
-    roboter.inboxLoop();
-    roboter.startStream();
-  } catch (error) {
-    logger.error(error);
+async function startBot(botConfig) {
```

Comment:
`async` kannst du vlt weglassen

Reply:
Scheint so, habe es lokal getestet,  ohne dass die Bots nun geantwortet haben, aber alle kamen zu dem Punkt "not replying because you are developing"